Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -298,3 +291,21 @@ extension Module  {
         return buildSettings
     }
 }
+
+extension Plist {
+    // FIXME: This is only internal for unit testing.
+    /// Quotes the strings in the Plist.
+    func quoteStrings() -> Plist {
+        switch self {
+        case .string(let str):
+            return .string("\"" + str + "\"")
```

Comment:
Taking a step back, though, I guess I don't see the reason why there is this function at all when there is `serialize` for plists.  Wouldn't this be quoting things twice?

Build settings that are arrays are a different thing.  Is that what this is used for?